spec/features/users/questions/upvotes-downvotes/user_can_vote_on_comments_on_answer_to_questions_spec.rb

```diff
@@ -16,15 +16,7 @@
     visit question_path(question)
 
     within("#answer-comment-upvote") do
-      expect(page).to have_button("Upvote")
-    end
-
-    within("#answer-comment-downvote") do
-      expect(page).to have_button("Downvote")
-    end
-
-    within("#answer-comment-upvote") do
-      click_button("Upvote")
+      find("[name=button]").click
```

Comment:
Since we changed the button from update to a material-icon this is clicking the button with the name button. Since we nicely separated our css selectors we can call the selector for that particular vote and then click the button with this.